scripts/pantheon/deploy

```diff
@@ -41,11 +41,11 @@ then
   if [[ "$CANONICAL_ENV" != "dev" ]]
   then
     terminus -n env:clone-content "$TERMINUS_SITE.$CANONICAL_ENV" dev
-    if [[ ! -z "$SANATIZE_SCRIPT" ]]
```

Comment:
Since this Env Variable is only used here, it's likely you can add the backwards compatibility here.

The gist will be to set `SANITIZE_SCRIPT="$SANATIZE_SCRIPT"` before any processing happens.

Reply:
You will also need to go the other way around... in case there are custom scripts that use the old variable.

Reply:
Actually you may want to put it in here: 
https://github.com/fourkitchens/pots/blob/1b8a89e79e390245a5bd43f946d0faa1848e9858/scripts/set-environment#L103-L152

At 103, add your logic to go one way... and then the other to make sure both are set to the same value.
Then echo them in the outputted bash script below after line 151

Test this on https://github.com/fourkitchens/test-project-ci

Reply:
just an update. I [got some logic in place](https://github.com/fourkitchens/pots/pull/44/files#diff-8aa36d0b89fbb5d5ae118f4d97a858ba74fb3aecfb4aee4abb97699a20a4dd15R104-R111), not entirely sure if it is working yet. It took me awhile to sort out how to set up testing a branch from a different repo. I now have circleci spinning up without failing so now I need to test if values are set and if the logic will work.

Reply:
You can at least see here that the empty string is being passed to the exported variable: https://app.circleci.com/pipelines/github/fourkitchens/test-project-ci/60/workflows/36ec40ed-a34e-4544-88b7-4b267409d145/jobs/201/parallel-runs/0/steps/0-107

Reply:
So you should be able to test your backward compatibility by setting one and then the other in your config.yml on test-project_ci. 

You can commit any script to test_projectci  as the script to be run... example you might create a script like ./.circleci/scripts/sanitize and have it do something like:
```
 #!/bin/bash

set -eo pipefail
echo "                               .                                                "
echo "                              %     (%.                                         "
echo "                              #          %.                                     "
echo "                              &.            @                                   "
echo "                            @                 ,(                                "
echo "                           #                     %                              "
echo "                          .,                      ./                            "
echo "                           *.                       /                           "
echo "                        ,&                          %                           "
echo "                     *#                            (                            "
echo "                   &                                    *&#                     "
echo "                  &                                          #                  "
echo "                 ,.                                           #                 "
echo "                  /  .&.     .%*               ,&.     .%*    &                 "
echo "                  @(.            %           (.            % #                  "
echo "              *@, @    .@@@@@*    @         @     @@@@@,    @@@/                "
echo "           %     (    /@@@@@@@@    %       (    *@@@@@@@&    %     #,           "
echo "          &      %    @@@@@@@@@    %       %    @@@@@@@@@    %       (          "
echo "         /       (    ,@@@@@@@#    #       (    *@@@@@@@%    #        @         "
echo "          /       @     @@@@@     @         @     @@@@@.    @         @         "
echo "           (       /,            #           /,            #         *          "
echo "          *#&        .&*     .&,               .@,     .&,          %.          "
echo "     #/                                                              .*&(       "
echo "  .(                   ,@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@                    %,   "
echo " &                       #                          #.                        @ "
echo "&                          ,%                    *#                            *"
echo "@                              ,@#.         ,&(                                 "
echo "/                                                                              *"
echo " %                                                                           .& "
echo "   #                     ...                                              (#    "
echo "     ,@(     *%@(,                       .,*(&&%/.               *%&#*          "
```
This would be test data you couldn't miss being run.

Then First add `SANITIZE_SCRIPT: ./.circleci/scripts/sanitize` to your config.yml and verify that you see this image when you deploy to Pantheon. Then remove the last line and add `SANATIZE_SCRIPT: ./.circleci/scripts/sanitize` and verify you still see the output but also your depreciation warning.